sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_administration_client.hpp

```diff
@@ -95,11 +95,28 @@ namespace Azure { namespace Security { namespace Attestation {
     Response<Models::AttestationToken<std::string>> GetAttestationPolicy(
         Models::AttestationType const& attestationType,
         GetPolicyOptions const& options = GetPolicyOptions(),
-        Azure::Core::Context const& context = Azure::Core::Context::ApplicationContext) const;
+        Azure::Core::Context const& context = Azure::Core::Context{}) const;
```

Comment:
As far as I have investigated, it is the same to do `Azure::Core::Context{}` than `Azure::Core::Context()`.

I wonder if we might want to set the same style for all the team?  expression list vs initializer list?  

For Context, we had been using expression list all over the place for the default constructor.  Would you propose to change everywhere to use curly braces?  Or would you just let it be as a free style choice?

Reply:
When in doubt, always check the [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax). Although, to be fair, I didn't think to check until you asked this question.

In this case, the guidance is unambiguous: The `{}` initializer syntax is preferred. IMHO we should follow that guidance whenever possible.

Reply:
Oh, and one other thought: `Foo()` can either be initialization of a `Foo` object, or a call to the method named `Foo`. `Foo{}` is unambiguous - it is always initialization of a Foo object.

I suspect that's a large part of why the guidelines prefer the `{}` initializer syntax (although the discussion section of the guidelines is empty :(). We an always ask Herb if it's important :).